db/migrate/20210903203639_drop_carts_table.rb

```diff
@@ -0,0 +1,4 @@
+class DropCartsTable < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
@ciejj zastanawiam się jak to Ci umknęło gdy sprawdzałeś ten PR przed wysłaniem. Bo sprawdzałeś, prawda? ;)

Reply:
to jest bardzo dobre pytanie ;)